.gitignore

```diff
@@ -6,11 +6,11 @@ protoc/protoc.zip
 protoc/readme.txt
 .install.tools
 proto/**/.done
-proto/v1beta1/*_go_proto/
```

Comment:
Why only include v1beta1? What about v1?

Reply:
This is because the Grafeas server currently only uses `v1beta1` protos. Yes, we'll add `v1` go-generated protos once the server has moved to using the new protos.